src/components/list/list.jsx

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    children: React.PropTypes.any,
+    type: React.PropTypes.oneOf(['ordered', 'unordered', 'description', 'icon'])
+  },
+
+  render: function() {
+    if (this.props.type === 'ordered') {
+      return (
+        <ol className="component-ordered-list">
+          {this.props.children}
```

Comment:
Sounds good! Will look into this.
List-items should accept any child, but `<dl>`, `<ul>` and `<ol>` should only accept the right children